environment-pipelines/iam.tf

```diff
@@ -736,6 +747,17 @@ data "aws_iam_policy_document" "opensearch" {
       "arn:aws:es:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:domain/*"
     ]
   }
+
+  statement {
+    actions = [
+      "es:ListVersions"
+    ]
+    effect = "Allow"
+    resources = [
+      "*"
+    ]
+    sid = "AllowOpensearchListVersions"
```

Comment:
As the above comment regarding "*"

Reply:
No since its not querying a resource, at most a condition could be added to tie it down but it isn't listing any of our data either way.